cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -7,7 +7,15 @@ class HttpCache
 
   # Language header and cookie are needed to separately cache language-specific pages.
   LANGUAGE_HEADER = %w(Accept-Language)
-  LANGUAGE_COOKIES = %w(language_ pm)
+
+  DEFAULT_COOKIES = [
+    # Language drop-down selection.
+    'language_',
+    # Page mode, for A/B experiments and feature-flag rollouts.
+    'pm',
```

Comment:
this PR doesn't change anything related to the use of this `pm` cookie (other than adding an explanatory comment), so the behavior of this cookie should not change at all here.

To answer/clarify your question about whether `pm` is 'per-user':

- **No**, the `pm` cookie value does not (and should not) contain a unique value for each individual user. (Using unique values in a cache-key makes the cached pages nearly useless, so I wouldn't recommend it.)
- **Yes**, you could write application code to set/modify the `pm` cookie value for individual user-agent/s, for example, by setting `pm` cookie to equal `test_header` for a few select test users on sign-in, so that only those specific users would see a special `test_header` page variant.